### ISSUE TITLE
Provide G1GC as the garbarge collector

### ISSUE THREAD

**Issue by santanusinha** [reines/dropwizard-debpkg-maven-plugin#26]
The jvm settings file forces ParNewGC, with the new G1GC in place, and with java 7 being end of life, maybe it makes sense to do one of the following:
- Set G1 as the default garbage collector
- Allow options to switch the garbage collector

**Comment by reines** [comment]
I agree. I'm happy to merge a PR if you can submit one, otherwise I suspect it'll be quite a while until I find time to look in to it myself I'm afraid.
